open_spiel/python/tests/policy_test.py

```diff
@@ -52,33 +52,17 @@
 
 class DerivedPolicyTest(absltest.TestCase):
```

Comment:
It would also be good to add a test that the derived policy can be used from C++ code.

Reply:
Should be no problem. Is there a simple (c++) function or class in the codebase that accepts a Policy pointer and can be called from python?

Would this be enough?

```
def test_cpp_policy_from_py(...):

     class DerivedPolicy(pyspiel.Policy):
            ...

     policy_obj = DerivedPolicy()

     that_one_pybound_cpp_func(policy_obj, potential_other_args_needed)
```

Reply:
added the test with calls for each method. It revealed that I had to actually use PYBIND11_OVERRIDE_NAME to get the correct calls from c++ as well!